src/applications/caregivers/utils/validation.js

```diff
@@ -27,6 +27,26 @@ export const validateCaregivers = (errors, _, formData) => {
   }
 };
 
+export const validateCountyInput = (errors, fieldData) => {
+  const disallowList = [
+    'US',
+    'U.S',
+    'U.S.',
+    'USA',
+    'U.SA',
+    'U.S.A',
+    'U.S.A.',
+    'United States',
+    'UnitedStates',
+    'United States of America',
+    'UnitedStatesOfAmerica',
+  ];
+  const regex = new RegExp(`^(?!.*(${disallowList.join('|')})).*$`, 'i');
```

Comment:
I wonder if this would be a better check to BE case sensitive? I know in Ohio we have `Tuscarawas` county which would fail this current validation because of the `US` in the list. If it was only capitalized it'd be a little more resilient. Or do we remove `US` from the list and keep the case insensitive? As you noted earlier I think this is what makes this type of validation pretty tricky.

Reply:
Yup.... This is why I absolutely HATE this idea and implementation.

Reply:
Decision made: we will retain "us" and "usa" as allowable, as those letter groups are included many county names. The rest will remain and are safe to be case insensitive.